ckanext/protected_resources/templates/package/snippets/lock_resource_button.html

```diff
@@ -1,4 +1,5 @@
-{% if resource.is_protected %}
+{% set is_protected = h.resource_has_protected_status(resource.id) %}
+{% if is_protected %}
```

Comment:
0/10
Would it be possible to do:
```{% if h.resource_has_protected_status(resource.id) %}```
instead?

Reply:
yes